web-console/src/dialogs/spec-dialog/spec-dialog.tsx

```diff
@@ -65,6 +65,7 @@ export const SpecDialog = React.memo(function SpecDialog(props: SpecDialogProps)
           tabSize: 2,
         }}
         style={{}}
+        placeholder="{ JSON spec... }"
```

Comment:
What does `placeholder` do?

Reply:
There are two changes in this PR that are thematically similar (when you fix an issue it is better to look for other potential issues in the code). A placeholder is the ghost text that appears in an input that goes away as soon as you start typing.

![image](https://user-images.githubusercontent.com/177816/89692848-a1057a80-d8c1-11ea-95c5-8c721ca45b09.png)

Before this change this dialog (which is not part of the data loader) would start of with `{\n\n}` as the initial value.
Now it start off blank (but with a placeholder) meaning that you do not need to delete or select over the initial value if you want to paste your own spec.